src/elements/common/annotator-context/withAnnotations.tsx

```diff
@@ -129,16 +131,40 @@ export default function withAnnotations<P extends object>(
             }
         };
 
+        handleAnnotationsFetchError = ({ error }: { error: Error }) => {
+            this.fetchError = error;
+        };
+
         handleAnnotator = (annotator: Annotator): void => {
```

Comment:
Should this be combined with `handleAnnotatorCreate`? The logic within seems like it could work as the result of either event.

Reply:
Do we even need to listen for the `annotator` event anymore in `ContentPreview`? Can we just use the new `annotator_create`?

Reply:
`handleAnnotatorCreate` needs to happen right after `annotator` construction (`init` has not finished yet), but hiding annotations has to happen after `init`. So both are needed.